catalog-info.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: mattermost-cloud-database-factory
```

Comment:
this is preference, still not sure where to end end up. The name could be just `Database Factory` or `Cloud Database Factory`. There's no need to match the repo name